recipes/funcsigs/meta.yaml

```diff
@@ -0,0 +1,31 @@
+{% set name="funcsigs" %}
+{% set version="1.0.2" %}
```

Comment:
About versions: I wonder if there's a way to have this package's version depend on whatever version it is effectively wrapping.  It seems like it will get annoying, and potentially worse (unresolvable dependencies somehow?)